App.js

```diff
@@ -12,16 +12,17 @@ import styles from './static/styles'
 
 const AppStack = createStackNavigator(
   { AppLoading: AppLoadingScreen, App: AppScreen },
-  { initialRouteName: 'AppLoading' }
+  { initialRouteName: 'AppLoading', headerMode: 'none' }
```

Comment:
Oh cool, that was it !?

Reply:
I also [see `headerBackTitleVisible`](https://reactnavigation.org/docs/en/stack-navigator.html).
Is it useless in our case then?

Reply:
indeed 😄